README.md

```diff
@@ -0,0 +1,210 @@
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Doctors Appoinment Backend ](#-doctors-appoinment-backend-)
+  - [🏡 Frontend App ](#-frontend-app-)
+  - [🚀 Kanban Board ](#-kanban-board-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Tests 🧪](#tests-)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Doctors Appoinment Backend <a name="about-project"></a>
+
+**[Doctors Appoinment Backend]** is a the backend part of a web application project that allows users to book appointments with doctors.
+<br></br>
+##  🏡 Frontend App <a name="front-end"></a>
+- > Link to 🎋 Doctors-Appoinments-Front-End [click here](https://github.com/adarachel/doctors-appointment-frontend)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Kanban Board <a name="kanban-board"></a>
+- [Kanban Board ](https://github.com/Enning94/Doctors_Appointment_Backend/projects/1)
+- Kanban Board Initial Screenshot
+![Kanban Board Initial Screenshot](<Screenshot 2023-10-01 at 10.10.19.png>)
+ 
+- We are grup of 5 people working on this project. We are using the Kanban board to keep track of our progress. We are using the following columns:
+  - Back_End-Team To Do
+  - Front_End-Team To Do
+  - in Progress
+  - in Review
+  - Done
+  
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org">Ruby On Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- *access the data using API*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+ ## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://health-book-doc.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby on Rails >=7](https://rubyonrails.org/)
+- [PostgreSQL](https://www.postgresql.org/)
+- [Git](https://git-scm.com/downloads)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:Enning94/Doctors_Appointment_Backend.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  bundle install
+```
```

Comment:
- [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate, rails db:seed``. 👍

Reply:
Thanks for reviewing the Pull Request, I will make the required changes🙂.